README.md

```diff
@@ -21,3 +21,58 @@ composer install
 Now you can go to http://demo.yii2-multilingual.dev/ and test multilingual stuff.
 
 For admin user password is 'admin'.
+
+## City location widget
+
+For correct working city widget must be usage [sypex-geo-daemon](https://github.com/DevGroup-ru/sypex-geo-daemon) or 
```

Comment:
Change to:
`City location widgets need correct working geo provider configured. For using [sypex-geo-daemon](https://github.com/DevGroup-ru/sypex-geo-daemon) configure yout settings in config/web.php:`